x-pack/plugin/esql/qa/server/multi-clusters/src/javaRestTest/java/org/elasticsearch/xpack/esql/ccq/MultiClusterSpecIT.java

```diff
@@ -96,7 +101,16 @@ public MultiClusterSpecIT(
         CsvTestCase testCase,
         String instructions
     ) {
-        super(fileName, groupName, testName, lineNumber, convertToRemoteIndices(testCase), instructions);
+        super(
+            fileName,
+            groupName,
+            testName,
+            lineNumber,
+            testCase.requiredCapabilities.contains(SUBQUERY_IN_FROM_COMMAND.capabilityName())
```

Comment:
I wonder if it wouldn't be cleaner to move this check to `convertToRemoteIndices`? This avoids duplicating location check, and since we pass `testCase` there shouldn't be a problem to check it.